tests/_dummies/model.py

```diff
@@ -12,36 +12,45 @@ def __init__(
         in_channels: int,
         num_classes: int,
         num_estimators: int,
+        dropout_rate: float,
     ) -> None:
         super().__init__()
+        self.dropout_rate = dropout_rate
 
         self.linear = nn.Linear(
             1,
             num_classes,
         )
+        self.dropout = nn.Dropout(p=dropout_rate)
 
         self.num_estimators = num_estimators
 
+    def feats_forward(self, x: Tensor) -> Tensor:
```

Comment:
Is it necessary?

Reply:
It does not seem to be right now
It could become helpful in the future if we want to test methods that work on the features, but I can remove it if you prefer.

Reply:
Okay I see, we can leave it here then.